addons/medical_gui/functions/fnc_updateActions.sqf

```diff
@@ -43,15 +43,17 @@ private _idcIndex = 0;
 {
     _x params ["_displayName", "_category", "_condition", "_statement"];
 
+    if (_forEachIndex < (GVAR(page)*9)) then {continue}; // Pagination
```

Comment:
The page size should be moved to define as it's used in multiple places.

Reply:
GVAR(page) is currently selected page, so this is calculating number of actions that are in the "history" of pages. i.e page 3 will have 18 actions that have been skipped across page 1 and 2, so skips first 18 iterations